dbld/images/alpine.dockerfile

```diff
@@ -0,0 +1,36 @@
+
+FROM alpine:3.21
+ENV OS_DISTRIBUTION=ubuntu
+ENV OS_DISTRIBUTION_CODE_NAME=noble
+
+ARG ARG_IMAGE_PLATFORM
+ARG COMMIT
+ENV IMAGE_PLATFORM=${ARG_IMAGE_PLATFORM}
+LABEL COMMIT=${COMMIT}
+
+
+#RUN /dbld/builddeps update_packages
+#RUN /dbld/builddeps install_dbld_dependencies
+#RUN /dbld/builddeps install_apt_packages
+#RUN /dbld/builddeps install_debian_build_deps
```

Comment:
[nitpick]

dead code

Reply:
I would drop this patch from this series. I've added docker-builder and docker targets to dbld/rules and with that we can build production images locally now.